backend/.env

```
@@ -16,6 +16,7 @@
 
 ###> symfony/framework-bundle ###
 APP_ENV=dev
+APP_DEBUG=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_DEBUG key should go before the APP_ENV key

(UnorderedKey)